packages/webapp/utils/helpers.ts

```diff
@@ -0,0 +1,284 @@
+import isEqual from 'lodash/isEqual'
```

Comment:
It's from [nouns-builder apps/web/utils](https://github.com/ourzora/nouns-builder/blob/main/apps/web/src/utils/helpers.ts).

Reply:
Yep, they were using at least one function from this file in my PR, so I just took the whole file anticipating there will be more. I don't usually just copy/paste code blindly but since we are copying the contracts they wrote for us we kinda have to copy their code surrounding it to

Reply:
For the ease of porting the frontend over we can keep it as is for now and I'll remove or refactor as needed.